x-pack/solutions/security/plugins/security_solution/common/detection_engine/prebuilt_rules/diff/extract_rule_data_query.test.ts

```diff
@@ -61,12 +64,35 @@ describe('extract rule data queries', () => {
             meta: {
               negate: false,
               disabled: false,
-              type: 'phrase',
-              key: 'test',
-              params: {
-                query: 'value',
+            },
+            query: {
+              term: {
+                field: 'value',
               },
             },
+          },
+        ],
+      });
+    });
+
+    it('normalizes filters without disabled field', () => {
+      const extractedKqlQuery = extractRuleKqlQuery(
+        'event.kind:alert',
+        'kuery',
+        [{ ...mockFilter, meta: { ...mockFilter.meta, disabled: undefined } }],
+        undefined
+      );
+
+      expect(extractedKqlQuery).toEqual({
+        type: KqlQueryType.inline_query,
+        query: 'event.kind:alert',
+        language: 'kuery',
+        filters: [
```

Comment:
In that test we are interested in `filters` field. Let's move `type`, `query` and `language` assertions to separate tests.